components/PostAddForm.js

```diff
@@ -3,33 +3,46 @@ import { useState, useCallback } from 'react';
 import Textarea from 'react-textarea-autosize';
 
 const PostAddForm = () => {
-  const { query: { offset } } = useRouter();
-  const [isExpanded, setExpanded] = useState(false);
-  const handleFocus = useCallback(() => setExpanded(true), []);
-  const handleCancel = useCallback(() => setExpanded(false), []);
+  const {query: {offset}} = useRouter();
 
-  if (offset)  {
+  if (offset) {
     return false;
   }
+  return <PostAddFormNotEmpty/>
+};
+
+const PostAddFormNotEmpty = () => {
+  const [isExpanded, setExpanded] = useState(false);
+  const handleFocus = useCallback(() => setExpanded(true), []);
+  const handleCancel = useCallback(() => setExpanded(false), []);
 
   return (
     <div>
-      <Textarea
-        minRows={3}
-        maxRows={10}
-        maxLength="1500"
-        defaultValue=""
-        onFocus={handleFocus}
-      />
-      {isExpanded && (
+      {!isExpanded ? (
+        <textarea
+          rows="3"
+          onFocus={handleFocus}
+        />
+      ) : <>
+        <Textarea
+          minRows={3}
+          maxRows={10}
+          maxLength="1500"
```

Comment:
Code style: {3}, {10}, "1500". Choose one. (Curly braces is the one, to pass down the numbers, not strings.)